src/code/z_play.c

```diff
@@ -960,25 +960,27 @@ void Play_UpdateMain(PlayState* this) {
                     break;
             }
         }
+
         Play_UpdateTransition(this);
+
         if (gTransitionTileState != TRANS_TILE_READY) {
-            if ((gSaveContext.gameMode == GAMEMODE_NORMAL) &&
-                (((this->msgCtx.msgMode == MSGMODE_NONE)) ||
-                 ((this->msgCtx.currentTextId == 0xFF) && (this->msgCtx.msgMode == MSGMODE_TEXT_DONE) &&
-                  (this->msgCtx.textboxEndType == TEXTBOX_ENDTYPE_41)) ||
-                 ((this->msgCtx.currentTextId >= 0x100) && (this->msgCtx.currentTextId <= 0x200))) &&
-                (this->gameOverCtx.state == GAMEOVER_INACTIVE)) {
+            if ((gSaveContext.gameMode == GAMEMODE_NORMAL &&
+                 ((this->msgCtx.msgMode == MSGMODE_NONE ||
+                   ((this->msgCtx.currentTextId == 0xFF && this->msgCtx.msgMode == MSGMODE_TEXT_DONE) &&
+                    (this->msgCtx.textboxEndType == TEXTBOX_ENDTYPE_41))) ||
+                  ((this->msgCtx.currentTextId >= 0x100) && (this->msgCtx.currentTextId <= 0x200)))) &&
```

Comment:
I think you took off brackets at the wrong spots. I would just copy what was there before, there should be no diff-change here:
```c
            if ((gSaveContext.gameMode == GAMEMODE_NORMAL) &&
                (((this->msgCtx.msgMode == MSGMODE_NONE)) ||
                 ((this->msgCtx.currentTextId == 0xFF) && (this->msgCtx.msgMode == MSGMODE_TEXT_DONE) &&
                  (this->msgCtx.textboxEndType == TEXTBOX_ENDTYPE_41)) ||
                 ((this->msgCtx.currentTextId >= 0x100) && (this->msgCtx.currentTextId <= 0x200))) &&
                (this->gameOverCtx.state == GAMEOVER_INACTIVE)) {
```